documentation/modules/exploit/multi/http/zabbix_script_exec.md

```diff
@@ -0,0 +1,81 @@
+## Vulnerable Application
+
+### Description
+
+Zabbix server allow remote command execution by two different way.
+
+The first way happen with the directive `AllowKey=system.run[*]` [[1]](https://blog.zabbix.com/zabbix-remote-commands/7500/#system.run).
+
+This directive is disabled by default, if an attacker already know Zabbix credentials 
+and the directive is enabled, he can abuse this functionnality to take control over the Zabbix server.
+
+The second way is by creating a script 
+[[2]](https://www.zabbix.com/documentation/devel/en/manual/web_interface/frontend_sections/administration/scripts) 
+in Zabbix and use it on a host.
+
+### Setup
+
+An easy way to setup a Zabbix server is to download and run their [appliance](https://www.zabbix.com/download_appliance).
+
+## Verification Steps
+
+Follow [Setup](#setup) and [Scenarios](#scenarios).
+
+## Targets
+
+### 0
+
+This uses a Linux dropper to execute code.
+
+### 1
+
+This executes a Unix command.
+
+## Options
+
+### USE_ITEM
+
+If this option is set, the exploit will try to use the key `systemn.run[*]` in an item, in order to achieve RCE.
+Otherwise, the exploit use a script on a host.
+
+### TLS_PSK
+Communication between an agent and the server can be encrypted with TLS using a PSK (pre-shared key).
+
+If the server is configured to enforce that, you have to get the key from the web interface (browse inside a host 
+and navigate to `Encryption` tab) and specifiy it with this option, otherwise RCE is not gonna happen.
+

```

Comment:
Can the module get this automatically?

Reply:
In some edge cases, the admin could have disabled the usage of TLS, but some host may still got the key registered, that's weird but that's technically possible if some host are not used anymore for example, I propose to notify the user of the module of existing key, and let him decide, what do you think ?